### PR TITLE
Adds new signage for the AI upload, AI sattelite, telecomms, vault, and R&D Servers

### PR DESCRIPTION
**Pull request by CoffeeDragon16**
## About The Pull Request
Reupload of #68194, I accidentally broke the branch. Hope that's the last time.

Adds new signs for the AI upload, AI sattelite, telecomms, vault, and R&D servers. Maps these in, shuffles things around to make them fit. 

## Why It's Good For The Game
 I was playing some other maps for the first time, and I noticed that if you're not familiar with the map it tends to be confusing what these rooms are at a glance.
More beginner friendly, and they're good atmosphere.

image image
image image
image image

## Changelog

:cl:
add:  New signage on all stations for the AI upload, AI sattelite, telecomms, vault, and the R&D servers.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/signs/signs_departments.dm [hunk 1]
```diff
@@ -209,6 +198,42 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/drop, 32)
...
+	name = "\improper AI Upload sign"
+	sign_change_name = "Location - AI Upload"
+	desc = "A sign labelling an area where laws are uploaded to the station's AI and cyborgs."
+	icon_state = "aiupload"
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/aiupload, 32)
+
+/obj/structure/sign/departments/aisat
+	name = "\improper AI Sattelite sign"
+	sign_change_name = "Location - AI Sattelite"
```
Comment (san7890) [inline code/game/objects/structures/signs/signs_departments.dm:227]:
```suggestion
	sign_change_name = "Location - AI Satellite"
```

Spellcheck

File: code/game/objects/structures/signs/signs_departments.dm [hunk 2]
```diff
@@ -209,6 +198,42 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/drop, 32)
...
+/obj/structure/sign/departments/aiupload
+	name = "\improper AI Upload sign"
+	sign_change_name = "Location - AI Upload"
+	desc = "A sign labelling an area where laws are uploaded to the station's AI and cyborgs."
+	icon_state = "aiupload"
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/aiupload, 32)
+
+/obj/structure/sign/departments/aisat
+	name = "\improper AI Sattelite sign"
```
Comment (san7890) [inline code/game/objects/structures/signs/signs_departments.dm:226]:
```suggestion
	name = "\improper AI Satellite sign"
```

Spellcheck

File: code/game/objects/structures/signs/signs_departments.dm [hunk 3]
```diff
@@ -209,6 +198,42 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/drop, 32)
...
+	sign_change_name = "Location - AI Sattelite"
+	desc = "A sign labelling a sattelite where the AI is installed."
```
Comment (san7890) [inline code/game/objects/structures/signs/signs_departments.dm:228]:
```suggestion
	desc = "A sign labelling a satellite where the AI is installed."
```

Spellcheck, also this just kinda reads odd to me? I dunno if there's a better way to phrase it, but I suppose this works.

File: code/game/objects/structures/signs/signs_departments.dm [hunk 4]
```diff
@@ -209,6 +198,42 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/drop, 32)
...
+	icon_state = "aisat"
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/aisat, 32)
+
+/obj/structure/sign/departments/vault
+	name = "\improper Vault sign"
+	sign_change_name = "Location- Vault"
```
Comment (san7890) [inline code/game/objects/structures/signs/signs_departments.dm:235]:
```suggestion
	sign_change_name = "Location - Vault"
```

Make the name formatting consistent.

File: code/game/objects/structures/signs/signs_departments.dm [hunk 5]
```diff
@@ -93,18 +86,24 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/science/alt, 32)
 	sign_change_name = "Department - Science: Xenobiology"
 	desc = "A sign labelling an area as a place where xenobiological entities are researched."
 	icon_state = "xenobio"
-	is_editable = TRUE
 
 MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/xenobio, 32)
 
+/obj/structure/sign/departments/rndserver
+	name ="\improper R&D  Server sign"
```
Comment (san7890) [inline code/game/objects/structures/signs/signs_departments.dm:93]:
```suggestion
	name ="\improper R&D Server sign"
```

Inconsistent spacing with the other names.

**Review by san7890 — CHANGES_REQUESTED** [review]
Few grammar/spelling/formatting oddities.

File: code/game/objects/structures/signs/signs_departments.dm (hunk 3 above)
Comment (CoffeeDragon16) [reply code/game/objects/structures/signs/signs_departments.dm:228]: It's consistent with the other sign's "A sign labelling an (area type) where (thing happens)" format. The intent of these messages is to give a short blurb on what the indicated area is to new players, who are the audience of these signs. However even if it breaks format it isn't very readable so I'll update it.

File: code/game/objects/structures/signs/signs_departments.dm [hunk 6]
```diff
@@ -209,6 +198,42 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/drop, 32)
...
+	sign_change_name = "Location - AI Upload"
+	desc = "A sign labelling an area where laws are uploaded to the station's AI and cyborgs."
+	icon_state = "aiupload"
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/departments/aiupload, 32)
+
+/obj/structure/sign/departments/aisat
+	name = "\improper AI Satellite sign"
+	sign_change_name = "Location - AI Satellite"
+	desc = "A sign labelling the AI's heavily-fortifited satellite."
```
Comment (san7890) [inline code/game/objects/structures/signs/signs_departments.dm:228]:
```suggestion
	desc = "A sign labelling the AI's heavily-fortified satellite."
```

Spellcheck :). Also, nicer description. Thanks!

**Review by Twaticus — APPROVED** [review]

**Comment by CoffeeDragon16** [comment]
Fuck

**Comment by san7890** [comment]
Alright, we aren't reverting stuff now so let's run this

**Pull request merged**